### DISCUSSION TITLE
Should we trim down the labels?

### DISCUSSION THREAD

**Discussion by clizbe** [TulipaEnergy/TulipaEnergyModel.jl#933] [Questions] [answered]
After using the "2 labels for every issue" system for a while - I don't think it's very useful.
I propose we remove all labels except the following:
- epic
- benchmark
- chore
- dependencies
- good first issue
- question
- stale
- bug? It's in the issue title as well, but maybe good to highlight them, especially for later on.

These all seem to have real purpose.
What do you guys think?

@datejada @gnawin @suvayu @abelsiqueira

**Comment by abelsiqueira** [comment, marked as answer]
+1 for removing

**Reply by clizbe** [reply]
Sold!

**Comment by suvayu** [comment]
Sorry, didn't respond earlier. The way I use labels are to filter issues/PRs. I didn't find the current system useful. I tend to filter for 2 reasons, maintenance tasks, or development. The smaller set helps for maintenance, but not development. In my projects I tend to add development related labels as required, but that doesn't mean it's littered with random labels. My label names tend to depend on the project. E.g. for TIO, I would probably use duckdb, sql, api, and maybe julia. It would be different for esdl4tulipa, maybe esdl, graph, influxdb, duckdb.

**Reply by clizbe** [reply]
Feel free to add those to TIO! 
I just wanted to remove the ones nobody was using.
(Sorry I removed all the zones/types in TIO so you'll have to relabel.)
